box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxJsonObject.java

```diff
@@ -24,8 +22,7 @@
 
     private static final long serialVersionUID = 7174936367401884790L;
     // Map that holds all the properties of the entity. LinkedHashMap was chosen to preserve ordering when outputting json
-    protected JsonObject mJsonObject;
-    transient CacheMap mCacheMap;
+    CacheMap mCacheMap;
```

Comment:
If this is not transient we need to change the private writ and read logic. Also not sure if we want to even bother restoring cached values.